.travis.yml

```diff
@@ -18,75 +17,137 @@ matrix:
+        - TOXENV=py27-nocov
+        - WHEEL_PATH=.tox/dist
+      python: '2.7'
+    - arch: amd64
+      env:
+        - TOXENV=py35-cover,codecov,extension-coveralls,coveralls
+      python: '3.5'
+    - arch: arm64
+      env:
         - TOXENV=py35-cover,codecov,extension-coveralls,coveralls
       python: '3.5'
-    - env:
+    - arch: amd64
+      env:
         - TOXENV=py35-nocov
         - WHEEL_MANYLINUX1=cp35
         - WHEEL_PATH=wheelhouse
       python: '3.5'
-    - env:
+    - arch: arm64
+      env:
+        - TOXENV=py35-nocov
+        - WHEEL_PATH=.tox/dist
+      python: '3.5'
+    - arch: amd64
+      env:
         - TOXENV=py36-cover,codecov,extension-coveralls,coveralls
       python: '3.6'
-    - env:
+    - arch: arm64
+      env:
+        - TOXENV=py36-cover,codecov,extension-coveralls,coveralls
+      python: '3.6'
+    - arch: amd64
+      env:
         - TOXENV=py36-nocov
         - WHEEL_MANYLINUX1=cp36
         - WHEEL_PATH=wheelhouse
       python: '3.6'
+    - arch: arm64
+      env:
+        - TOXENV=py36-nocov
+        - WHEEL_PATH=.tox/dist
+      python: '3.6'
     - os: osx
       language: generic
       env:
         - TOXENV=py37-cover
-    - env:
+    - arch: amd64
+      env:
+        - TOXENV=py37-cover,codecov,extension-coveralls,coveralls
+      python: '3.7'
+    - arch: arm64
+      env:
         - TOXENV=py37-cover,codecov,extension-coveralls,coveralls
       python: '3.7'
     - os: osx
       language: generic
       env:
         - TOXENV=py37-nocov
         - WHEEL_PATH=.tox/dist
-    - env:
+    - arch: amd64
+      env:
         - TOXENV=py37-nocov
         - WHEEL_MANYLINUX1=cp37
         - WHEEL_PATH=wheelhouse
       python: '3.7'
-    - env:
+    - arch: arm64
+      env:
+        - TOXENV=py37-nocov
+        - WHEEL_PATH=.tox/dist
+      python: '3.7'
+    - arch: amd64
+      env:
         - TOXENV=py38-cover,codecov,extension-coveralls,coveralls
       python: '3.8'
-    - env:
+    - arch: arm64
+      env:
+        - TOXENV=py38-cover,codecov,extension-coveralls,coveralls
+      python: '3.8'
+    - arch: amd64
+      env:
         - TOXENV=py38-nocov
         - WHEEL_MANYLINUX1=cp38
         - WHEEL_PATH=wheelhouse
       python: '3.8'
-    - env:
+    - arch: arm64
+      env:
+        - TOXENV=py38-nocov
+        - WHEEL_PATH=.tox/dist
+      python: '3.8'
+    - arch: amd64
+      env:
         - TOXENV=pypy-cover,codecov,extension-coveralls,coveralls
       python: 'pypy'
-    - env:
+    - arch: amd64
+      env:
         - TOXENV=pypy-nocov
       python: 'pypy'
-    - env:
+    - arch: amd64
+      env:
         - TOXENV=pypy3-cover,codecov,extension-coveralls,coveralls
         - TOXPYTHON=pypy3
       python: 'pypy3'
-    - env:
+    - arch: amd64
+      env:
         - TOXENV=pypy3-nocov
         - TOXPYTHON=pypy3
       python: 'pypy3'
 before_install:
+  - if [[ ${TRAVIS_CPU_ARCH} != "arm64" ]]; then 
+        export LD_PRELOAD=/lib/x86_64-linux-gnu/libSegFault.so;
```

Comment:
Could this maybe be `/lib/aarch64-linux-gnu/libSegFault.so`? Is there a way to verify that (eg, you have a arm64 machine around)?

Reply:
I have checked in my local machine and added as suggested. Please review and let me know if anymore changes required.